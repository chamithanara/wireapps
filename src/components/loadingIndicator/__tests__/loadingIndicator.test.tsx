import React from 'react';
import { render, screen } from '@testing-library/react-native';
import LoadingIndicator from '..';

describe('LoadingIndicator', () => {
    test('matches the snapshot', () => {
        render(<LoadingIndicator />);
        expect(screen.toJSON()).toMatchSnapshot();
    });
});
