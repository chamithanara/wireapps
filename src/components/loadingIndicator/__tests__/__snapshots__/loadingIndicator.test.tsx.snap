// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadingIndicator matches the snapshot 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "alignSelf": "center",
      "flexDirection": "column",
      "height": 50,
      "justifyContent": "center",
      "width": "100%",
    }
  }
>
  <ActivityIndicator
    color="#FFFFFF"
    size={25}
  />
</View>
`;
